generators/app/constants.js

```diff
@@ -80,7 +96,7 @@ exports.files = [
   {
     directory: 'migrations/migrations',
     name: '.keep',
-    condition: answers => answers.orm && answers.orm.sequelize
+    condition: answers => answers.orm && answers.orm.sequelize && answers.technology === 'nodeJS'
```

Comment:
i'd change technology name to "expressJS"

Reply:
ok